### DISCUSSION TITLE
Authorization Override For specific controller to specific action.

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/aspnetcore#28723] [General]
Hi,

I have a scenario where Whole of my system (Based on Asp.net core 3.1) is using Same Authorized Filter policy.
So we have added in our startup as a global filter.
We are using JWT scheme in application.

Now I have a requirement for new controller with Whole together new policy. Global filter should not affect that controller actions.

If any one have any idea to overcome this scenario. We can discuss in this thread.

**Comment by Tratcher** [comment]
Rather than using a global filter you should use the FallbackPolicy on AuthorizationOptions to apply auth to everything. Then you should apply the Authorize attribute with a specific policy to your new controller.
